source/what/trunk-repos.html.md

```diff
@@ -26,7 +26,7 @@ For stable releases, we also follow the upper-constraints approach.
 
 ## Hashed repos and special repos
 
-When each new upstream commit is built, a repository is created with the new package and the latest versions from every other package built by RDO Trunk. This new repository is located in a hashed URL, for example [https://trunk.rdoproject.org/centos7/51/29/51299ba1eca22462c5560854f53b668ad7872ac7_18d76a1f/](https://trunk.rdoproject.org/centos7/51/29/51299ba1eca22462c5560854f53b668ad7872ac7_18d76a1f/). This hash is built using:
+When each new upstream commit is built, a repository is created with the new package and the latest versions from every other package built by RDO Trunk. This new repository is located in a hashed URL. The details can be found at [https://www.rdoproject.org/what/trunk-repos/](https://www.rdoproject.org/what/trunk-repos/). This hash is built using:
```

Comment:
Altough the link is broken the point of this section is to explain the format of the url for hashed repos, and i think it's useful to keep it. We purge repos older that 60 days so it's difficult to keep updated info. Anyway, IMO the fact that the link is not as important as understanding what each part of the url means.

Reply:
If the URL is an explanation, then we should not create a clickable link. We can use backslashes to create inline code, so we can show the URL without having the Markdown parser create a link for it.